src/mdoc/Verifier.ts

```diff
@@ -157,21 +167,21 @@ export class Verifier {
     }
 
     if (deviceAuth.deviceSignature) {
-      const deviceKey = await importCOSEKey(deviceKeyCoseKey);
+      const deviceKey = await COSEKey.import(deviceKeyCoseKey).toKeyLike();
```

Comment:
please note that `toKeyLike()` is still calling to jose's `importJWK` method:

https://github.com/jfromaniello/cose-kit/blob/afc207099f4a3fc957209faf32010bbc8c34f713/src/key/COSEKey.ts#L152-L153